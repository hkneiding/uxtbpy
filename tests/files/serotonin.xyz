25

C          1.53037        1.32549        0.01696
C          0.18424        1.69571        0.01972
C         -0.76990        0.67054        0.00083
C         -0.43059       -0.68441       -0.02309
C          0.93779       -1.03400       -0.02354
C          1.89414       -0.01421       -0.00328
O          3.22988       -0.29822       -0.00070
C         -1.65577       -1.43142       -0.04308
C         -2.68492       -0.50898       -0.02331
N         -2.14236        0.74545        0.00236
C         -1.81930       -2.92158       -0.07925
C         -1.91028       -3.45791       -1.51358
N         -2.10946       -4.90841       -1.54366
H          2.30055        2.09293        0.03271
H         -0.10408        2.74116        0.03758
H          1.22836       -2.07786       -0.03596
H          3.33701       -1.26338       -0.00138
H         -3.75873       -0.64388       -0.02468
H         -2.66768        1.60839        0.02195
H         -2.72909       -3.19367        0.47254
H         -0.98385       -3.39788        0.44893
H         -0.99916       -3.21036       -2.07027
H         -2.74752       -2.98194       -2.03697
H         -1.28637       -5.37603       -1.16592
H         -2.18863       -5.21244       -2.51339
