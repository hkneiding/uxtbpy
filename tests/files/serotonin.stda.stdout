2025-7-23 14:5:18.801


                 *********************************************
                 *                                           *
                 *               s  T  D  A                  *
                 *                                           *
                 *                S. Grimme                  *
                 * Mulliken Center for Theoretical Chemistry *
                 *             Universitaet Bonn             *
                 *              Version 1.6.3                *
                 *       Fri Aug 26 14:28:49 CEST 2022       *
                 *********************************************

 Please cite as:
 S. Grimme, J. Chem. Phys. 138 (2013) 244104
 M. de Wergifosse, S. Grimme, J. Phys. Chem A
 125 (2021) 18 3841-3851

 With contributions from:
 C. Bannwarth, P. Shushkov, M. de Wergifosse

======================================================================

 reading an xTB output...

 reading: wfn.xtb
======================================================================
                    M O / A O    I N P U T
======================================================================
atom  #          x             y             z            charge
  C   1      2.89197990    2.50481285    0.03204975         6.00
  C   2      0.34816311    3.20442719    0.03726540         6.00
  C   3     -1.45490001    1.26713684    0.00156847         6.00
  C   4     -0.81369710   -1.29334734   -0.04363377         6.00
  C   5      1.77216610   -1.95397663   -0.04448415         6.00
  C   6      3.57940551   -0.02685301   -0.00619830         6.00
  O   7      6.10358805   -0.56355407   -0.00132281         8.00
  C   8     -3.12895154   -2.70499152   -0.08140939         6.00
  C   9     -5.07376300   -0.96183271   -0.04404951         6.00
  N  10     -4.04847328    1.40869621    0.00445975         7.00
  C  11     -3.43797842   -5.52098554   -0.14976078         6.00
  C  12     -3.60990569   -6.53450226   -2.86025140         6.00
  N  13     -3.98630130   -9.27554975   -2.91709436         7.00
  H  14      4.34740903    3.95506413    0.06181294         1.00
  H  15     -0.19668268    5.18004118    0.07101590         1.00
  H  16      2.32126377   -3.92658596   -0.06795455         1.00
  H  17      6.30603439   -2.38744197   -0.00260782         1.00
  H  18     -7.10296962   -1.21675674   -0.04663844         1.00
  H  19     -5.04118412    3.03941632    0.04147948         1.00
  H  20     -5.15723219   -6.03516107    0.89297110         1.00
  H  21     -1.85920688   -6.42106201    0.84835467         1.00
  H  22     -1.88813858   -6.06670060   -3.91224294         1.00
  H  23     -5.19205984   -5.63504940   -3.84931507         1.00
  H  24     -2.43088677  -10.15922340   -2.20326928         1.00
  H  25     -4.13591090   -9.85008313   -4.74961830         1.00

 # atoms          =   25
 # mos            =  111
 # primitive  aos =  848
 # contracted aos =  164

 spherical AO basis
======================================================================
                    A O   I N T E G R A L S
======================================================================
 memory needed for AO int data:   1 Mb
 done.

Restricted MOs found... RKS-sTDA will be performed

======================================================================
                    s T D A
======================================================================
 spectral range up to (eV)     :    7.00000000000000
 occ MO cut-off (eV)           :   -25.7600292614343
 virtMO cut-off (eV)           :    5.54030682669241
 perturbation thr              :   1.000000000000000E-004
 triplet                       :  F
 MOs in TDA :           62
 oMOs in TDA:           24
 vMOs in TDA:           38
 sorting MOs ...
 reading and transforming R..V..L AO ints ...
 ints done.
 S^1/2 ...
 S^1/2 orthogonalized MO coefficients done.

 SCF atom population (using active MOs):
  2.913  2.894  2.557  2.794  2.829  2.123  3.695  2.887  2.674  3.136
  3.118  3.055  4.239  0.780  0.795  0.796  0.454  0.759  0.579  0.801
  0.804  0.830  0.830  0.839  0.818

 # electrons in TDA:  48.000


 ax(DF)   :   0.500000000000000
 s^K      :    2.00000000000000
 beta  (J):    4.00000000000000
 alpha (K):    2.00000000000000

 hardness table read.
 setting up gammas ...
 memory needed for q data (Mb)                      0
 computing q(ij,n) ...

          14 CSF included by energy.

         898 considered in PT2.
 ordered frontier orbitals
         eV     # centers
  14   -20.361    12.2
  15   -20.164    10.1
  16   -19.797    11.8
  17   -19.537     8.0
  18   -18.765    10.3
  19   -18.361    14.8
  20   -17.627     8.1
  21   -16.498     6.6
  22   -15.969     2.2
  23   -14.726     6.0
  24   -14.060     6.3

  25    -6.160     6.2
  26    -4.913    20.2
  27    -4.613    12.2
  28    -4.587     6.7
  29    -4.372    19.1
  30    -3.923    18.2
  31    -3.820    10.3
  32    -3.685    17.3
  33    -3.615    18.8
  34    -3.191    12.4
  35    -2.883    17.7

             lowest CSF states
       eV     nm      excitation i->a               eV
    1  4.72   262.4       24 ->  25     gap,J,K:   7.900   4.591   0.708   Kshft:   0.000  locality: 0.983
    2  5.32   233.1       23 ->  25     gap,J,K:   8.566   4.686   0.720   Kshft:   0.000  locality: 1.000
    3  5.55   223.4       24 ->  26     gap,J,K:   9.147   4.114   0.008   Kshft:   0.500  locality: 0.956
    4  5.67   218.5       24 ->  27     gap,J,K:   9.447   4.316   0.070   Kshft:   0.405  locality: 0.976
    5  5.74   216.1       24 ->  28     gap,J,K:   9.473   4.582   0.422   Kshft:   0.002  locality: 0.981
    6  6.18   200.6       24 ->  29     gap,J,K:   9.687   4.033   0.013   Kshft:   0.500  locality: 0.961
    7  6.31   196.4       23 ->  27     gap,J,K:  10.113   4.292   0.087   Kshft:   0.316  locality: 0.967
    8  6.38   194.3       23 ->  26     gap,J,K:   9.813   3.949   0.009   Kshft:   0.500  locality: 0.914
    9  6.58   188.5       23 ->  28     gap,J,K:  10.139   4.669   0.553   Kshft:   0.001  locality: 0.996
   10  6.64   186.8       21 ->  25     gap,J,K:  10.338   4.721   0.509   Kshft:   0.001  locality: 0.992
   11  6.64   186.7       24 ->  31     gap,J,K:  10.239   4.365   0.383   Kshft:   0.002  locality: 0.976
   12  6.80   182.5       24 ->  30     gap,J,K:  10.137   3.854   0.006   Kshft:   0.500  locality: 0.942
   13  6.80   182.4       24 ->  33     gap,J,K:  10.445   4.121   0.093   Kshft:   0.287  locality: 0.953
   14  6.85   181.0       23 ->  29     gap,J,K:  10.354   4.035   0.016   Kshft:   0.500  locality: 0.958

 selecting CSF ...
 average/max PT2 energy lowering (eV):     0.036     0.086
         123 CSF included by PT.
         137 CSF in total.
 calculating TDA matrix ...
 diagonalizing ...
 estimated time (min)     0.00
   17 roots found, lowest/highest eigenvalue :    4.192   6.940   0
  perform velo correction for X...
  writing trafoed spectral data to tda.dat ...

 excitation energies, transition moments and TDA amplitudes
 state    eV      nm       fL        Rv(corr)
    1    4.192   295.8     0.1227    -0.7021     0.86(  24->  25) -0.33(  23->  25) -0.20(  23->  28)
    2    4.487   276.3     0.0750    -6.7958     0.74(  23->  25)  0.51(  24->  28)  0.39(  24->  25)
    3    5.095   243.4     0.0068     4.3698    -0.64(  24->  26)  0.63(  24->  27) -0.26(  24->  28)
    4    5.282   234.7     0.0286    13.9238     0.56(  24->  26)  0.47(  24->  27)  0.35(  24->  29)
    5    5.709   217.2     0.0121     0.3282     0.54(  23->  26) -0.48(  24->  29) -0.46(  23->  27)
    6    5.808   213.5     0.0209    22.5492     0.53(  23->  27) -0.47(  24->  29)  0.39(  24->  26)
    7    5.958   208.1     0.4527    -8.2596    -0.58(  24->  28) -0.40(  24->  27)  0.36(  23->  25)
    8    6.083   203.8     0.0057   -12.9833    -0.53(  24->  31)  0.47(  24->  30) -0.31(  23->  26)
    9    6.159   201.3     0.0119    14.0932     0.49(  23->  28)  0.47(  21->  25) -0.36(  23->  26)
   10    6.346   195.4     0.0105    -0.3912    -0.51(  24->  33)  0.46(  23->  29)  0.32(  23->  27)
   11    6.466   191.7     0.0229   -17.2549     0.61(  22->  26) -0.41(  22->  29) -0.41(  22->  30)
   12    6.631   187.0     0.3176    16.7272    -0.38(  24->  32)  0.36(  23->  26)  0.29(  23->  27)
   13    6.655   186.3     0.0332   -34.2540    -0.42(  24->  33) -0.35(  23->  33)  0.35(  23->  30)
   14    6.776   183.0     0.0970   -11.7399     0.41(  24->  30) -0.37(  23->  28)  0.33(  24->  31)
   15    6.816   181.9     0.6049   -15.6247    -0.63(  24->  32)  0.25(  23->  25)  0.25(  24->  31)
   16    6.904   179.6     0.3039    23.4381    -0.49(  21->  25)  0.40(  23->  28)  0.28(  20->  25)
   17    6.940   178.7     0.1480   -31.6883     0.48(  23->  29) -0.33(  23->  31) -0.32(  23->  26)

  alpha tensor

           1         2         3

    1  89.47625
    2  19.48166  41.53345
    3   1.56491   1.49083   1.30622
 trace alpha_L[0] / au:   44.1053061558602
 sum rule f_L           2.27423303876174

 SOS specific optical rotation
 including Lorentz factor for common solvent (n=1.4)
 lambda [eV] alpha[grad*cm^3*g^-1*dm^-1]
 632.8  1.96     -232.96
 589.3  2.10     -272.39        0.00 ##
 579.0  2.14     -283.26
 546.0  2.27     -323.09
 436.0  2.84     -551.72
 365.0  3.40     -917.71

 sTDA done.
2025-7-23 14:5:18.999
